main.tf

```diff
@@ -44,3 +44,9 @@ resource "aws_guardduty_member" "members" {
   invite             = "${lookup(var.member_list[count.index], "invite")}"
   invitation_message = "please accept guardduty invitation"
 }
+
+resource "aws_guardduty_invite_accepter" "member_accepter" {
+  count             = "${var.accept_invite && (length(var.accepter_master_account_id) > 0) ? 1 : 0}"
+  detector_id       = "${aws_guardduty_detector.master.id}"
+  master_account_id = "${var.accepter_master_account_id}"
+}
```

Comment:
Actually this module is meant for master guardduty which can invite other member accounts. This above code changes also works, if we want to make this account to be invited from other master account.
I'll update the readme accordingly..

Reply:
The module has nothing that makes it master only. You've already account for the setup of a member account's detector inherently due to the count on the member list. 

I use this module for both sides of the fence.

I'll update the README - apologies I should have done so already!